app/src/main/java/com/duckduckgo/app/onboarding/ui/page/WelcomePage.kt

```diff
@@ -57,6 +81,23 @@ class WelcomePage : OnboardingPageFragment() {
         typingAnimation?.cancel()
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
I know this is an experiment but I feel this page could have a view model so this logic could leave there and be tested ala `DefaultBrowserPageViewModel`. 

How about creating a clean up task in Asana to create that view model, test it, etc? If we end up deciding to release to everyone we can do that so, otherwise we delete the task.

Reply:
Done